### ISSUE TITLE
Lighthouse Audit - start_url does not respond with a 200 when offline.

### ISSUE THREAD

**Issue by wesdse001** [preactjs/preact-cli#1517]
**Do you want to request a _feature_ or report a _bug_?**

Bug

**What is the current behaviour?**

After deploying a built application and running a Lighthouse audit via Chrome, the following warning appears in the Progressive Web App section of the audit:

```
start_url does not respond with a 200 when offline. The start_url did respond, but not via a service worker.
```

🎆🎆🎆 100 scores in all the other categories , thank you Preact team!!!

And the Progressive Web App does work and installs on a mobile phone. However, I am just wondering how I can fix the above warning.

**If the current behavior is a bug, please provide the steps to reproduce.**

```
preact build --no-esm --no-prerender --no-inline-css
```

I also tried just `preact build` and get the same warning.

Please paste the results of `preact info` here.

```
Environment Info:
  System:
    OS: macOS 11.0
    CPU: (12) x64 Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
  Binaries:
    Node: 12.14.1 - ~/.nvm/versions/node/v12.14.1/bin/node
    npm: 6.14.8 - ~/.nvm/versions/node/v12.14.1/bin/npm
  Browsers:
    Chrome: 83.0.4103.97
    Safari: 14.0.1
  npmPackages:
    preact: 10.5.7
    preact-cli: 3.0.5 
    preact-render-to-string: 5.1.12 
    preact-router: 3.2.1
```

**Comment by StAlRuth** [comment]
I can second this issue, including that my PWA works offline and when installed.

**Comment by wesdse001** [comment]
This issue has been fixed in the latest version of Lighthouse `7.0.0`.
